src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -261,6 +261,16 @@ public interface IDurableOrchestrationClient
         /// <returns>Returns a task which completes when the status has been fetched.</returns>
         Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId, bool showHistory = false, bool showHistoryOutput = false, bool showInput = true);
 
+        /// <summary>
+        /// Get the status of multiple instances.
+        /// </summary>
+        /// <param name="instanceIds"> List of multiple instance IDs that needed to check that status.</param>
+        /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
+        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
+        /// <param name="showInput">If set, fetch and return the input for the orchestration instance.</param>
+        /// <returns>Returns a list of orchestration status values.</returns>
+        Task<List<DurableOrchestrationStatus>> GetStatusAsync(IEnumerable<string> instanceIds, bool showHistory = false, bool showHistoryOutput = false, bool showInput = false);
```

Comment:
In case you decide to take my previous suggestion on `DurableClient.cs`, then you'll need to change the return value here to be `IList` instead of `List`.

```suggestion
        Task<IList<DurableOrchestrationStatus>> GetStatusAsync(IEnumerable<string> instanceIds, bool showHistory = false, bool showHistoryOutput = false, bool showInput = false);
```